js/apps/system/_admin/aardvark/APP/frontend/js/views/databaseView.js

```diff
@@ -1,513 +0,0 @@
-      var toDelete = this.collection.where({name: dbname});
-      toDelete[0].destroy({wait: true, url: arangoHelper.databaseUrl('/_api/database/' + encodeURIComponent(dbname))});
-      this.updateDatabases();
-      window.App.naviView.dbSelectionView.render($('#dbSelect'));
-      window.modalView.hide();
-    },
-
-    changeDatabase: function (e) {
-      var changeTo = $(e.currentTarget).attr('id');
-      var url = this.collection.createDatabaseURL(changeTo);
-      window.location.replace(url);
-    },
-
-    updateDatabases: function () {
-      var self = this;
-      this.collection.fetch({
-        cache: false,
-        success: function () {
-          self.render();
-          window.App.handleSelectDatabase();
-        }
-      });
-    },
-
-    editDatabase: function (e) {
-      var dbName = this.evaluateDatabaseName($(e.currentTarget).attr('id'), '_edit-database');
-      var isDeletable = true;
-      if (dbName === this.currentDB) {
-        isDeletable = false;
-      }
-      this.createEditDatabaseModal(dbName, isDeletable);
-    },
-
-    search: function () {
-      var searchInput,
-        searchString,
-        strLength,
-        reducedCollection;
-
-      searchInput = $('#databaseSearchInput');
-      searchString = arangoHelper.escapeHtml($('#databaseSearchInput').val().normalize());
-      reducedCollection = this.collection.filter(
-        function (u) {
-          return u.get('name').indexOf(searchString) !== -1;
-        }
-      );
-      $(this.el).html(this.template.render({
-        collection: reducedCollection,
-        searchString: searchString,
-        currentDB: this.currentDB,
-        readOnly: this.readOnly
-      }));
-      this.replaceSVGs();
-
-      // after rendering, get the "new" element
-      searchInput = $('#databaseSearchInput');
-      // set focus on end of text in input field
-      strLength = searchInput.val().length;
-      searchInput.focus();
-      searchInput[0].setSelectionRange(strLength, strLength);
-    },
-
-    replaceSVGs: function () {
-      $('.svgToReplace').each(function () {
-        var img = $(this);
-        var id = img.attr('id');
-        var src = img.attr('src');
-        $.get(src, function (d) {
-          var svg = $(d).find('svg');
-          svg.attr('id', id)
-            .attr('class', 'tile-icon-svg')
-            .removeAttr('xmlns:a');
-          img.replaceWith(svg);
-        }, 'xml');
-      });
-    },
-
-    remove: function () {
-      this.$el.empty().off(); /* off to unbind the events */
-      this.stopListening();
-      this.unbind();
-      delete this.el;
-      return this;
-    },
-
-    evaluateDatabaseName: function (str, substr) {
-      var index = str.lastIndexOf(substr);
-      return str.substring(0, index);
-    },
-
-    getDatabaseSettings: function(dbName) {
-      return $.ajax({
-        type: 'GET',
-        cache: false,
-        url: arangoHelper.databaseUrl('/_api/database/current', _.escape(dbName)),
```

Comment:
are we planning to use the default values from this API? I don't see this API being used anymore